benchmarks/timeseries/resampling.py

```diff
@@ -46,7 +46,7 @@ def _do_work() -> None:
             for _n_sample in range(samples):
                 now = now + timedelta(seconds=1 / samples)
                 helper.add_sample(Sample(now, Quantity(0.0)))
-            helper.resample(now)
+            helper.resample(now, helper._name)  # pylint: disable=protected-access
```

Comment:
I just realized you're calling `helper.resample`, so from within `resample`, you can just do `self._name` and don't have to pass it from the outside.

Reply:
Good that you found it out, I'll amend it accordingly

Reply:
Updated